lib/peep.rb

```diff
@@ -0,0 +1,10 @@
+class Peep
+
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :peep_text, String
+  property :username, String
+  property :name, String
+
```

Comment:
This should probably `belong_to` a User.

Then, you can ask for all a User's Peeps with `@user.peeps`, and the user who owns a particular Peep with `@peep.user`.